### PR TITLE
UI

### PR DESCRIPTION
**Pull request by garshaam**

### CODE REVIEW

File: gtkfunctions.cpp [hunk 1]
```diff
@@ -507,14 +594,16 @@ static void show_spectrum(GtkWidget *widget, GdkEventButton *event, gpointer dat
   void * data_new2=img_struct1->HyperFunctions1;
   HyperFunctions *HyperFunctions1=static_cast<HyperFunctions*>(data_new2);
 
-
+    cout <<"hi";
```
Comment (antmedellin) [inline gtkfunctions.cpp:597]: please remove

File: gtkfunctions.cpp [hunk 2]
```diff
@@ -507,14 +594,16 @@ static void show_spectrum(GtkWidget *widget, GdkEventButton *event, gpointer dat
...
  // show the spectrum here 
   // https://github.com/opencv/opencv_contrib/blob/master/modules/plot/samples/plot_demo.cpp 
     // https://docs.opencv.org/4.x/d0/d1e/classcv_1_1plot_1_1Plot2d.html
     
     Mat data_x( 1, HyperFunctions1->mlt1.size(), CV_64F ); // wavelength
     Mat data_y( 1, HyperFunctions1->mlt1.size(), CV_64F ); // reflectance value
 
+    std::cout << "whatever" << std::endl;
```
Comment (antmedellin) [inline gtkfunctions.cpp:605]: please remove

File: image_tool.cpp [hunk 3]
```diff
@@ -17,8 +17,16 @@ using namespace std;
...
+
+  struct spin_struct {
+  GObject *button1;
+  GObject *button2;
+  GObject *button3;
+  HyperFunctions *HyperFunctions1;
+  } ;
   
-  
+  //cubert hyperspectral has their band info online
```
Comment (antmedellin) [inline image_tool.cpp:28]: please remove these comment

File: image_tool.cpp [hunk 4]
```diff
@@ -84,6 +72,8 @@ int main (int argc, char *argv[])
 
   entry_struct *gtk_hyper_entry, temp_var3;
   gtk_hyper_entry=&temp_var3;
+  //Why does gtk_hyper_entry get used for both of the following buttons
+  //What is the point of temp_vars
```
Comment (antmedellin) [inline image_tool.cpp:76]: here

**Review by antmedellin — APPROVED** [review]

**Review by alextran1144 — APPROVED** [review]
Looks good

**Pull request merged**